sessionctx/variable/noop.go

```diff
@@ -128,7 +128,7 @@ var noopSysVars = []*SysVar{
 	{Scope: ScopeNone, Name: "skip_networking", Value: "0"},
 	{Scope: ScopeGlobal, Name: "innodb_monitor_reset", Value: ""},
 	{Scope: ScopeNone, Name: "ssl_cipher", Value: ""},
-	{Scope: ScopeNone, Name: "tls_version", Value: "TLSv1,TLSv1.1,TLSv1.2"},
+	{Scope: ScopeNone, Name: "tls_version", Value: getRuntimeTLSVersion()},
```

Comment:
With this change this isn't really a noop anymore. Maybe we should move this to another file.
Just something we need to consider, if we decide to keep it here that's fine with me, but we need to make an explicit choice here.

Reply:
How about moving it to `defaultSysVars`

Reply:
Yes that would be good.